bin/p2-exec/main.go

```diff
@@ -80,20 +82,31 @@ func main() {
 		}
 	}
 
+	if *podID != "" {
+		cgConfig, err := cgroups.GetPodConfig(pods.ResourceLimitsConfigPathEnvVar)
+		if err != nil {
+			log.Fatal(err)
+		}
+		err = cgroups.CreatePodCgroup(*podID, cgConfig)
+		if err != nil {
+			log.Fatalf("Could not create pod %v cgroup %v\n", *podID, err)
+		}
+	}
+
 	if *launchableName == "" && *cgroupName != "" {
 		log.Fatalf("Specified cgroup name %q, but no launchable name was specified", *cgroupName)
```

Comment:
is it intended that you'll have already created the pod cgroup here? would it be better to validate all the input first or does it not matter